config/zones/AL.yaml

```diff
@@ -5,50 +5,56 @@ bounding_box:
     - 42.6882473822
 capacity:
   biomass:
-    - datetime: '2023-01-01'
+    - datetime: '2018-01-01'
       source: entsoe.eu
       value: 0.0
   coal:
-    - datetime: '2023-01-01'
+    - datetime: '2021-01-01'
       source: entsoe.eu
       value: 0.0
   gas:
-    - datetime: '2024-01-01'
+    - datetime: '2021-01-01'
       source: entsoe.eu
       value: 0.0
   geothermal:
-    - datetime: '2024-01-01'
+    - datetime: '2021-01-01'
       source: entsoe.eu
       value: 0.0
   hydro:
-    - datetime: '2023-01-01'
+    - datetime: '2018-01-01'
+      source: entsoe.eu
+      value: 1492.0
+    - datetime: '2021-01-01'
       source: entsoe.eu
       value: 2203.0
   hydro storage:
-    - datetime: '2024-01-01'
+    - datetime: '2021-01-01'
       source: entsoe.eu
       value: 0.0
   nuclear:
-    - datetime: '2023-01-01'
+    - datetime: '2021-01-01'
       source: entsoe.eu
       value: 0.0
   oil:
-    - datetime: '2023-01-01'
+    - datetime: '2021-01-01'
```

Comment:
@VIKTORVAV99 is this correct? Seems weird to change the date back 2 years?

Reply:
Yes, this capacity format basically only record the date of the first time that value has been seen and then that's valid until the next value.

So if the capacity for a mode is:
0 for year 1
2 for year 2
2 for year 3
4 for year 4

Then the you can skip the year 3 in the config as the year 2 value will be valid until year 4.